config/requirements_core.txt

```diff
@@ -1,7 +1,7 @@
 # The necessary dependencies to run Tabbycat
 
 # Django
-Django==3.1.1
+Django==3.2.1
```

Comment:
Huh, Django 3.2.1 doesn't seems to exist for me and it says [expected May 4, 2021](https://docs.djangoproject.com/en/3.2/releases/3.2.1/). Was this meant to be Django 3.2?